Makefile

```diff
@@ -461,8 +461,9 @@ fetch-clang-format: .FORCE
 	$(E) ".clang-format"
 	$(Q) scripts/fetch-clang-format.sh
 
+BASE ?= "HEAD~1"
 indent:
-	find . -name '*.[ch]' -type f -print0 | xargs --null --max-args 128 --max-procs 4 clang-format -i
+	git clang-format --style file --extensions c,h --quiet $(BASE) || true
```

Comment:
why do we need true here?

Reply:
The git-clang-format exits with non-zero exit code if it finds mistakes in code, if we remove `|| true` we have:

```
[snorch@turmoil criu]$ git diff
diff --git a/Makefile b/Makefile
index e055adcac..af3888e79 100644
--- a/Makefile
+++ b/Makefile
@@ -463,7 +463,7 @@ fetch-clang-format: .FORCE
 
 BASE ?= "HEAD~1"
 indent:
-       git clang-format --style file --extensions c,h --quiet $(BASE) || true
+       git clang-format --style file --extensions c,h --quiet $(BASE)
 .PHONY: indent
 
 include Makefile.install
[snorch@turmoil criu]$ make indent BASE=008c2b9c7f86a5e27fc721a28e3669134ef98969
git clang-format --style file --extensions c,h --quiet 008c2b9c7f86a5e27fc721a28e3669134ef98969
make: *** [Makefile:466: indent] Error 1
```
I decided to hide this error but didn't find any flag of git-clang-format itself to do it.

Reasons for hiding:
- One of usecases of `make indent` I believe is to check or fix format in newly created patch. In this case it is pretty much expected that git-clang-format would fix something and printing error each time may be missleading.
- We used to check git-diff after make indent anyways (e.g. in lint.yml) printing our custom nicely formated error anyways.

Reply:
pls add a comment that explains this

Reply:
Oops, I read the comment again and I think we don't need to hide the error if it finds mistakes. I expects it exits with a non-zero code in this case.  And I think we don't need the --quiet option here.

> One of usecases of make indent I believe is to check or fix format in newly created patch. In this case it is pretty much expected that git-clang-format would fix something and printing error each time may be missleading.

It depends how to look at this, but I understand your concerns. I would suggest to add the --diff option instead of hiding the error code.

Reply:
> I would suggest to add the --diff option instead of hiding the error code.

```
--diff                print a diff instead of applying the changes
```
If we use diff option we would need to apply changes manually. Is it what we want?

Reply:
See updated variant, hope it would be ok.